src/main/deploy/auton/BluePodiumWing1Center0.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE auton SYSTEM "auton.dtd">
+<auton>
+
+<primitive id = "RESET_POSITION_PATH_PLANNER"
+pathname= "BluePodiumWing1Center0.path"/>
+
+<snippet file = "ShootWhenStop.xml"/>
+
+<primitive id= "DRIVE_PATH_PLANNER"
+pathname = "BluePodiumWing1Center0.path"/>
```

Comment:
remove .path from the file name

add zones for switching states example ...


    <primitive id= "DRIVE_PATH_PLANNER"
            pathname="BlueSubwooferWing1Center0">
            <zone filename="BlueIntakeWingNote2.xml"/>
    </primitive>